assets/js/modules/analytics-4/utils/is-zero-report.js

```diff
@@ -34,11 +34,13 @@ export function isZeroReport( report ) {
 		return undefined;
 	}
 
-	// If there is no rows array, no totals array, or totals is an array of empty objects, this is an empty report.
+	// If there is no rows array, no totals array, the totals row is boolean (RE: #8442),
+	// or totals is an array of empty objects, this is an empty report.
 	if (
 		! report?.rows ||
 		! report?.totals ||
-		report?.totals?.every( isEmpty )
+		typeof report?.totals !== 'object' ||
```

Comment:
This additional check is redundant, as `report?.totals?.every?.( isEmpty )` already effectively handles this, resulting in condition being `false` if `totals` is anything other than `Array`